CIP-0072/README.md

```diff
@@ -0,0 +1,613 @@
+            "type": "string",
+            "description": "An optional long dApp description (no less than 40 and no longer than 1008 characters).",
+            "minLength": 40,
+            "maxLength": 1008
+          }
+        },
+        "required": [
+          "short"
+        ]
+      },
+      "releases": {
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "releaseNumber": {
+                "type": "string",
+                "pattern": "^(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(-((0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(\\.(0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\\+([0-9a-zA-Z-]+(\\.[0-9a-zA-Z-]+)*))?$",
+                "description": "Semver compatible release number (major.minor.patch) or (major.minor.patch-some_text) e.g. 1.2.3 or 1.1.1-alpha"
+              },
+              "releaseName": {
+                "type": "string",
+                "description": "An optional human readable release name, e.g. V1"
+              },
+              "securityVulnerability": {
+                "type": "boolean",
+                "description": "Indicates that a given version has a security vulnerability."
+              },
+              "comment": {
+                "type": "string",
+                "description": "A free text field to provide comment about this particular release, e.g. new features it brings, etc."
+              },
+              "scripts": {
+                "type": "array",
+                "items": [
+                  {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string"
+                      },
+                      "version": {
+                        "type": "integer"
+                      }
+                    },
+                    "required": [
+                      "id",
+                      "version"
+                    ]
+                  }
+                ]
+              }
+            },
+            "required": [
+              "releaseNumber"
+            ]
+          }
+        ]
+      },
+      "scripts": {
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties":{
+              "id": {
+                "type":"string",
+                "description": "Unique Script ID (across all scripts from this dApp)."
+              },
+              "name":{
+                "type":"string",
+                "description": "An optional script name usually related to it's function."
+              },
+              "purposes":{
+                "type":"array",
+                "items": {
+                    "type": "string",
+                      "enum":["SPEND", "MINT"]
+                },
+                "description": "Purpouses of the script, SPEND or MINT (notice it can be both for some modern Cardano languages)."
```

Comment:
```suggestion
                "description": "Purposes of the script, SPEND or MINT (notice it can be both for some modern Cardano languages)."
```
This misspelling happens a few times :)